src/shared/components/adminTabs/FlashRound/FlashRoundAdmin.tsx

```diff
@@ -143,15 +143,24 @@ const FlashRoundAdmin = () => {
       });
   };
 
+  var currentThemeSong = new Audio(round_start)
+  var playingCurrentThemeSong = false
   const playSound = (audioPath: string, timeout: number) => {
-    const audio = new Audio(audioPath);
-    const { play, stop } = useSounds(audio);
-    play(audio);
-    setTimeout(() => {
-      stop(audio);
-    }, timeout);
-    return audio;
-  };
+    const audio = new Audio(audioPath)   
+    const {play, stop} = useSounds(audio)    
+    play(audio)    
+    playingCurrentThemeSong = true
```

Comment:
usually in React we try to avoid variable assignment, make use of the setter function returned from the useState hook